### PR TITLE
feat(icons): add checkmark and exclamation mark icons LS-3819

### PR DESCRIPTION
**Pull request by ryanjdiercks**
## Jira

https://littlespoon.atlassian.net/browse/LS-3819

## Motivation

Add checkmark and exclamation icons in icons package

### CODE REVIEW

File: packages/icons/src/CheckIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+}: CheckIconProps) {
+  return (
+    <CheckIconBase
+      width="24"
+      height="24"
+      viewBox="0 0 24 24"
+      size={size}
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+      {...other}
```
Comment (fjallkraken) [inline packages/icons/src/CheckIcon.tsx:36]: Do we want to support the override of the previous props? If that's not the case, it'll be better to move the spreading before the explicit props.

Comment (remarkablemark) [reply packages/icons/src/CheckIcon.tsx:36]: I think we want to override other props

**Review by remarkablemark — APPROVED** [review]

**Comment by remarkablemark** [comment]
@ryanjdiercks can you amend your commit since this should be a `feat` rather than a `build` (a `build` should be when you update package dependencies):

```sh
git commit --amend "feat(icons): add checkmark and exclamation mark icons LS-3819"
git push -f origin feat/LS-3819
```

**Comment by ryanjdiercks** [comment]
> @ryanjdiercks can you amend your commit since this should be a `feat` rather than a `build` (a `build` should be when you update package dependencies):
> 
> ```shell
> git commit --amend "feat(icons): add checkmark and exclamation mark icons LS-3819"
> git push -f origin feat/LS-3819
> ```

I updated the commit message.

**Review by remarkablemark — APPROVED** [review]

**Pull request merged**